README.md

```diff
@@ -1,70 +1,52 @@
-# Getting Started with Create React App
+# Leaderboard
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Additional description about the project and its features:
```

Comment:
- [ ] Please add a descriptive project description to make your Readme more informative.